pkg/platform/model/vcs.go

```diff
@@ -600,13 +602,37 @@ func CommitInitial(hostPlatform string, langName, langVersion string) (strfmt.UU
 	return res.Payload.CommitID, nil
 }
 
+var versionRe = regexp.MustCompile(`^\d+(\.\d+)+$`)
```

Comment:
Consider making the repeat non-capturing, and limited in count: `^\d+(?:\.\d+){0,2}$`
Also, how will other variations play into this? Like `3.4.5rc2` or `1.2.3x4`.
Maybe something more specific to the wildcards:
```
func isExactVersion(v string) bool {
	return !strings.HasSuffix(v, ".x") && !strings.Contains(v, ".x.")
}
```